crates/meilisearch/tests/common/server.rs

```diff
@@ -327,7 +327,7 @@ impl<State> Server<State> {
         self.service.get(url).await
     }
 
-    pub(super) fn _index(&self, uid: impl AsRef<str>) -> Index<'_> {
+    pub fn _index(&self, uid: impl AsRef<str>) -> Index<'_> {
```

Comment:
@irevoire not sure if this is intended, making an underscore-starting method public is tingling my spider sense

Reply:
Didn't like it too but removing it forces code to be moved to `common` which I didn't like either